apps/site/assets/css/_footer.scss

```diff
@@ -58,38 +133,244 @@
+          display: block;
+        }
+      }
+    }
+
+    @include media-breakpoint-down(sm) {
+      padding-top: 16px;
+      padding-bottom: 14px;
+      a {
+        margin-top: 20px;
+      }
+    }
+
+    @include media-breakpoint-down(xs) {
+      margin-bottom: 0px;
+      padding-top: 24px;
+    }
+
+    @include media-breakpoint-up(lg) {
+      padding-bottom: 26px;
+    }
+
+  small {
+    font-size: 16px;
+    @include media-breakpoint-down(xs) {
+      white-space: nowrap;
+    }
+  }
+  
+  }
+
+  @include media-breakpoint-up(lg) {
+    font-size: $font-size-base;
+    padding-left: 120px;
+  }
+
+  @include media-breakpoint-only(md) {
+    font-size: $font-size-base-sm;
+    padding-bottom: $base-spacing * 2;
   }
 
-  .m-footer--mobile__items {
-    text-align: center;
+  @include media-breakpoint-down(sm) {
+    padding-bottom: 50px;
+  }
+  .m-footer__links {
+    @include media-breakpoint-down(xs){
+      display: block;
+    }
+    @include media-breakpoint-up(lg){
+      display: block;
 
-    @include not-last-child {
-      &::after {
-        color: $white;
-        content: '●';
-        margin-left: .25em;
-        margin-right: .25em;
+    }
+  }
+  @include media-breakpoint-up(lg) {
+    margin-bottom: 0px;
+    padding-right: 40px;
+    .container {
+      ul {
+        display: flex;
+        flex-direction: column;
+        .m-footer__group {
+          width: 100%;
+        }
       }
     }
+
+    .m-footer__group {
+      padding-left: 15px;
+    }
   }
 
-  .m-footer--mobile__item {
-    display: inline;
+  :nth-child(2) {
+    :nth-child(1) {
+      margin-top: 0;
+    }
+  }
+
+  .m-menu__feature_phone {
+    padding-top: 1rem;
+    @include media-breakpoint-down(xs) {
+      display: flex;
+      a {
+        padding-top: 0px;
+        margin-top: 0px;
+      }
+      strong {
+        padding-right: 4px;
+      }
+    }
+  }
+  @include media-breakpoint-down(lg) {
+    a {
+      padding-top: 12px;
+    }
   }
+}
 
-  .m-footer__copyright {
-    padding-top: 0;
-    text-align: center;
+.call-us-header {
+  @include media-breakpoint-up(lg) {
+    padding-left: 12px;
+    font-size:23px;
+    margin-bottom: 12px;
+    font-family: Inter UI,Helvetica Neue,Helvetica,Arial,sans-serif;
+  }
+  @include media-breakpoint-down(sm) {
+    padding-left: 15px;
+  }
+  @include media-breakpoint-down(xs) {
+    margin-top: 24px;
+    font-size: 18px;
+    margin-bottom: 8px;
+  }
+}
+
+.numbers-container {
+  padding-top: 19px;
+  .m-footer__item {
+    display: flex;
+  }
+  strong {
+    margin-top: 12px;
+    padding-right: 4px;
+  }
+  a {
+    color: $brand-primary;
+  }
+}
+
+.all-container {
+  @include media-breakpoint-up(lg) {
+    display: flex;
+  }
+}
+
+.lower-footer {
+  @include media-breakpoint-down(xs) {
+    flex-direction: column;
+  }
+
+  @include media-breakpoint-up(lg) {
+    position: absolute;
+    margin: 0 40px 0 40px;
+    left: 0;
+    right: 0;
+    bottom: 60px;
```

Comment:
**question:** See my other comment about avoiding using `position: absolute`. What it's needing to do here and is there a simpler way? For example, if it's main function is to push the lower footer further down the page, it could be more simply achieved by adding extra `margin-top`.